# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/basic/bug656261.js
# Commit: 7cda151d97a5
# Full Hash: 7cda151d97a53e47a1f526390dbb005ba8f78fca
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-10-28 21:22:11
# Regressor Bug: 1797755
# File Overlap Count: 1
# Description:
#   Bug 1797755 - Part 5: Use a single initial mark stack size regardless of whether incremental GC is enabled r=sfink
#   
#   Currently we initialize the mark stack to a different size depending on whether
#   or not incremental GC is enable. However, after the first GC we always shrink
#   it to the initial size when it is disabled.
# ==============================================================================

diff -r 098569654f17 -r 7cda151d97a5 js/src/jit-test/tests/basic/bug656261.js
--- a/js/src/jit-test/tests/basic/bug656261.js	Fri Oct 28 09:47:14 2022 +0000
+++ b/js/src/jit-test/tests/basic/bug656261.js	Fri Oct 28 09:47:15 2022 +0000
@@ -5,7 +5,7 @@
 
 function test()
 {
-    var N = internalConst("INCREMENTAL_MARK_STACK_BASE_CAPACITY") + 2;
+    var N = internalConst("MARK_STACK_BASE_CAPACITY") + 2;
     var o = {};
     var descriptor = { enumerable: true};
     for (var i = 0; i != N; ++i) {
